src/Protocol/ActivityPub/Transmitter.php

```diff
@@ -478,7 +478,7 @@ private static function createPermissionBlockForItem($item, $blindcopy, $last_id
 								$data['to'][] = $profile['url'];
 							} else {
 								$data['cc'][] = $profile['url'];
-								if (($item['private'] != Item::PRIVATE) && $item['private'] && !empty($actor_profile['followers'])) {
```

Comment:
Is that correct to include `$item['private']` at this point? I just wonder! :-) And yes, we need to implement ActivityPub correctly.

Reply:
What do you mean by "include `$item['private']`"?

Reply:
The check for it has been removed, means "private" items are included. I hope the logic here is correct.

Reply:
If you look exactly at the new code line, you see that `$item['private']` is now used this way: `($item['private'] != Item::PRIVATE)`.

Reply:
Okay, maybe next time I should take a closer look. ;-)